.github/workflows/test.yml

```diff
@@ -39,6 +39,9 @@ jobs:
           INFURA_PROJECT_SECRET: ${{ secrets.INFURA_PROJECT_SECRET }}
           EVM_RPC_URL: ${{ secrets.EVM_RPC_URL }}
           WALLET_PUBLIC_ADDRESS: ${{ secrets.WALLET_PUBLIC_ADDRESS }}
+          INFURA_IPFS_PROJECT_ID: ${{ secrets.INFURA_IPFS_PROJECT_ID }}
+          INFURA_IPFS_PROJECT_SECRET: ${{ secrets.INFURA_IPFS_PROJECT_SECRET }}
+          INFURA_IPFS_ENDPOINT: ${{ secrets.INFURA_IPFS_ENDPOINT }}
```

Comment:
IMO, `INFURA_IPFS_ENDPOINT` must be hardcoded. It will be always `https://ipfs.infura.io:5001/`, there is no choice and less confusing for the final user.

Reply:
OK for me